packages/utils/node-resolver-core/package.json

```
@@ -14,12 +14,22 @@
     "url": "https://github.com/parcel-bundler/parcel.git"
   },
   "main": "lib/NodeResolver.js",
```

Shouldn't make a different, but for consistency...
```suggestion
  "main": "lib/Wrapper.js",
```